docs/pages/tutorials/reduce-build-times-with-cache.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Reduce build times with remote caching
+sidebar_label: Reduce build times
+---
+
+Devpod provides the ability to use a registry as a remote backend cache for your container builds. To enable it perform
+
+```
+./devpod-cli context set-options -o REGISTRY_CACHE={registry}
+```
+
+where registry follows the syntax "\{domain\}/\{project\}/\{repo\}", such as gcr.io/my-project/my-dev-env
+
+
+Ensure your docker environment has the image snapshotter enabled in your daemon (https://docs.docker.com/engine/storage/containerd/)
```

Comment:
This is only required for local docker builds, right?

Reply:
yes that's correct, i'll update :)